openshift_sti_images/nodejs.adoc

```diff
@@ -0,0 +1,130 @@
+= NodeJS
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Our OpenShift https://github.com/openshift/sti-nodejs/tree/master/0.10[NodeJS] image repository contains the sources and Dockerfiles for building various versions of NodeJS platform images. The resulting images can be run either by Docker or using https://github.com/openshift/source-to-image[STI].
+
+== Versions
+Current supported versions of NodeJS:
+
+* https://github.com/openshift/sti-nodejs/tree/master/0.10[0.10]
+
+== Base Images
+
+* RHEL-7
+* CentOS-7
+
+[NOTE]
```

Comment:
Sure. I was looking into the merged code for this style, eg image_writers_guide/sti.adoc and was following the pattern.
Thanks for update !